docs/user-guides/acme-dns-validation.md

```diff
@@ -0,0 +1,114 @@
+# Issuing an ACME certificate using DNS validation
+
+`cert-manager` can be used to obtain certificates from a CA using the [ACME][1] protocol. The ACME protocol supports various challenges which are used to prove ownership of a domain so that a valid certificate can be issued for that domain. One such challenge is `dns-01`. With the DNS challenge, you prove you own a domain by proving you control its DNS records. This is done by creating a TXT record with specific content.
```

Comment:
`supports various challenge mechanisms`
`one such challenge mechanism is`
`With a DNS01 challenge, you prove ownership of a domain by demonstrating you control its DNS records.`

Reply:
I've changed it to dns01 when referring to the field and DNS-01 for everything else (same for http) but let me know if you want me to change that